datafusion/expr/src/partition_evaluator.rs

```diff
@@ -17,14 +17,24 @@
 
 //! Partition evaluation module
 
-use crate::window::window_expr::BuiltinWindowState;
-use crate::window::WindowAggState;
+use crate::window_frame_state::WindowAggState;
```

Comment:
This file is moved into `datafusion_expr` and I had to make a few small changes related to state management -- but I think I may be able to avoid that.